astropy/units/tests/test_quantity_ufuncs.py

```diff
@@ -290,7 +290,6 @@ def test_reciprocal_array(self):
         assert np.all(np.reciprocal(np.array([1., 2., 4.]) * u.m)
                       == np.array([1., 0.5, 0.25]) / u.m)
 
-    # cbrt only introduced in numpy 1.10
```

Comment:
Could you also remove the `skipif("not hasattr(np, 'cbrt'))` lines a bit further down?